cmd/mysterium_client/command_run/factory.go

```diff
@@ -7,16 +7,13 @@ import (
 	"github.com/mysterium/node/openvpn"
 	"github.com/mysterium/node/server"
 	dto_discovery "github.com/mysterium/node/service_discovery/dto"
-	"os"
 )
 
 func NewCommand(vpnMiddlewares ...openvpn.ManagementMiddleware) *CommandRun {
 	nats_discovery.Bootstrap()
 	openvpn.Bootstrap()
 
 	return &CommandRun{
-		Output:      os.Stdout,
```

Comment:
Why? :/

Reply:
Because its not used anywhere

Reply:
Why to keep unused variables anyway? Boy scout rule -> keep a place a bit cleaner than found.